core/inputs/input.ts

```diff
@@ -123,7 +123,6 @@ export class Input {
 
     if (this.sourceBlock.rendered) {
```

Comment:
would it be possible to just check `instanceof BlockSvg` then you wouldn't need to cast either? circular dependencies are the only reason i can think of you wouldn't do that but i'm hoping that doesn't apply here

Reply:
It causes circular dependencies.

Reply:
:(